src/Symfony/Component/Intl/Languages.php

```diff
@@ -137,6 +150,10 @@ public static function getAlpha3Names($displayLocale = null): array
         $alpha2Names = self::getNames($displayLocale);
         $alpha3Names = [];
         foreach ($alpha2Names as $alpha2Code => $name) {
+            if (3 === \strlen($alpha2Code)) {
+                $alpha3Names[$alpha2Code] = $name;
+                continue;
+            }
             try {
                 $alpha3Names[self::getAlpha3Code($alpha2Code)] = $name;
             } catch (MissingResourceException $e) {
```

Comment:
In this catch clause we should have the line:
```
  $alpha3Names[$alpha2Code] = $name;
```

Get Alpha3Names should not drop any names from the list.

Reply:
see https://github.com/symfony/symfony/pull/33140#discussion_r313785511

the lists are independent, we should not include alpha2 codes (2-letter at this point) in `getAlpha3Names`, so yes we drop those.

Reply:
If alpha2 codes includes localized languages (like Flemish), the the alpha3 codes should too.

In the catch the `$alpha2Code` variable would hold a longer than 3 code.

Reply:
which would be a locale most likely, not an alpha3 code. At this point we passed 3-letter codes as well as 2-letter codes with a mapping.

This could be further optimized in #33148 

`Flemish` is mapped `nl_BE`, but we want those out the list actually (hence #33148), but `getAlpha3Name` could account for e.g. `nld_BE` yes, this i will handle in #33148 :)